examples/user-management/nuxt3-user-management/README.md

```diff
@@ -8,17 +8,36 @@ This repo will demonstrate how to:
 - store and retrieve data with [Supabase database](https://supabase.io/docs/guides/database)
 - store image files in [Supabase storage](https://supabase.io/docs/guides/storage)
 
+
+## Technologies used
+
+- Frontend:
+  - [Nuxt.js 3](https://github.com/nuxt/nuxt) - The Intuitive Vue Framework.
+  - [Supabase.js](https://supabase.com/docs/library/getting-started) for user management and realtime data syncing.
+  - Supabase [Supabase module for Nuxt.](https://github.com/nuxt-modules/supabase).
+- Backend:
+  - [supabase.com/dashboard](https://supabase.com/dashboard/): hosted Postgres database with restful API for usage with Supabase.js.
+
+
 ## Getting Started
 
-Before running this app, you need to create a Supabase project and copy [your credentials](https://supabase.io/docs/guides/with-nuxt-3#get-the-api-keys) to `.env`.
+Iniate a new project in Nuxt 3 with `npx nuxi@latest init <project-name>`.
```

Comment:
After initing the project, the nuxi cli supports auto-installing the modules now as:
```bash
npx nuxi module add supabase
```
so you could make it as the step 2